cmd/run.go

```diff
@@ -107,6 +109,34 @@ a commandline interface for interacting with it.`,
 			Left:  func() string { return "    init" },
 		}
 
+		// Load plugins
```

Comment:
Moving this up from where it was.

Long story short: plugins need to be loaded before the runner gets initialized, as lookup paths are going to be resolved by that time. `getConfig` is the earliest point at which we have access to the plugin parameters so I decided to pull it up and do it here.

Reply:
As I mention in the other comment, you shouldn't use `getConfig()` here, but rather move and use `getRuntimeOptions()`